utils/data-migration/helpers/database.js

```diff
@@ -158,13 +156,17 @@ const tfmDealUpdate = async (updatedDeal) => {
  * @returns {Promise} Resolve if successful otherwise reject.
  */
 const tfmFacilityUpdate = async (updatedFacility) => {
+  if (typeof updatedFacility.facilitySnapshot.ukefFacilityId !== 'string') {
```

Comment:
Can we please use optional chaining there `updatedFacility?.facilitySnapshot?.ukefFactilityId...`